### DISCUSSION TITLE
Is it possible to integrate `marshmallow` or `pydantic`?

### DISCUSSION THREAD

**Discussion by YaoJusheng** [python-restx/flask-restx#628] [Q&A]
As per the question, is anyone currently working on or will be working on integration related to marshmallow or pydantic in order to better uniformly manage and define incoming and outgoing parameters and perform serialization operations and verification?

**Comment by peter-doggart** [comment]
`flask-restx` is only minimally maintained currently. It's unlikely any large changes like this will happen in the short term. There would likely also need to be a change for OpenAPI v3 for this to be easily supported. 

There is a fairly large discussion about this over on issues here: https://github.com/python-restx/flask-restx/issues/59

**Reply by YaoJusheng** [reply]
Thanks for your reply. I took a quick look and found that some developers have proposed an implementation example of introducing an external library to define a model and then converting it to a restx model. 

Are there any clear plans or are they currently being implemented?
